ios/RNGoogleSignin/RNGoogleSignin.m

```diff
@@ -51,11 +56,21 @@ - (instancetype)init {
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
+  NSString *path = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info" ofType:@"plist"];
+
+  if (!path) {
+    RCTLogError(@"RNGoogleSignin: Missing GoogleService-Info.plist");
+    reject(@"INTERNAL_MISSING_CONFIG", @"Missing GoogleService-Info.plist", nil);
+    return;
+  }
+
+  NSMutableDictionary *plist = [[NSMutableDictionary alloc] initWithContentsOfFile:path];
```

Comment:
small thing, but seems like this can be a `NSDictionary` (since it's not being mutated)